cmake/UthequeConfigVersion.cmake.in

```diff
@@ -0,0 +1,11 @@
+set(PACKAGE_VERSION "@robot_dart_VERSION@")
```

Comment:
We keep robot_dart version here?

Reply:
The alternative is to define a second version number, but do we want to keep things desynchronized?

Reply:
Let's keep one.. Agreed..